### PR TITLE
docs: add customer calls and infrastructure page

### PR DESCRIPTION
**Pull request by galexrt**

### CODE REVIEW

File: docs/consulting/customer-calls.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Customer Calls
+---
+
+# Customer Calls
+
+The obvious first, be respective, nice and confident in your communications.
```
Comment (dave-at-koor) [inline docs/consulting/customer-calls.md:7]: *respective -> respectful

File: docs/consulting/customer-calls.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+The obvious first, be respective, nice and confident in your communications. 
+
+Try to reiterate points brought up by the customer to make sure that everyone is on the same page.
+
+Should a customer have any questions that you can't answer on the spot, make sure to write them down and add them to the customer ticket as a ticket so it can be tracked.
+Same goes for topics a customer might have questions about, document them in a ticket so that they don't fall under the rug and get forgotten.
+
+## Discovery Call
+
+We use a simple "discovery call checklist" in form of a Google form here: [](https://docs.google.com/forms/d/1uModm46mQkSSWV84CNNL-_m5nnsX9yo58xkwxb--SYQ/edit?usp=forms_home&ths=true).
```
Comment (dave-at-koor) [inline docs/consulting/customer-calls.md:16]: need to put something between [] for link to show up. e.g., [Google form]

**Review by dave-at-koor — CHANGES_REQUESTED** [review]
Looks good. Just small issues.

**Comment by galexrt** [comment]
@dave-at-koor PTAL

**Review by dave-at-koor — APPROVED** [review]

**Pull request merged**